OnboardPietroPieri/App.tsx

```diff
@@ -69,36 +70,73 @@ const styles = StyleSheet.create({
 });
 
 type SectionProps = PropsWithChildren<{
+  type?: string,
   title: string;
+  setLogin: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
O tipo do setState é complexo assim porque ele permite outros usos que não são necessários aqui. Para o caso atual, é só uma função que recebe uma string, então daria para simplificar assim:
```suggestion
  setLogin: (value: string) => void;
```

Reply:
aplicado